packages/office-ui-fabric-react/src/components/SelectedItemsList/examples/SelectedPeopleList.Basic.Example.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from 'office-ui-fabric-react/lib/SelectedItemsList';
 import { Selection } from 'office-ui-fabric-react/lib/Selection';
 import { Stack } from 'office-ui-fabric-react/lib/Stack';
-import { people, groupOne, groupTwo } from '../../ExtendedPicker/examples/PeopleExampleData';
+import { people, groupOne, groupTwo } from 'office-ui-fabric-react/lib/components/ExtendedPicker/examples/PeopleExampleData';
```

Comment:
These example data imports are a little awkward just because they do deep imports which we explicitly discourage .. I don't remember exactly but do we not have an example utils package or the like?

Reply:
Hmm, an example data package could nicely solve a LOT of different problems... I'll add that to my list of possible approaches to consider.

Reply:
Downside is it would almost necessarily introduce circular dependencies: OUFR needs to depend on the example data package, which also needs interfaces from OUFR. :(

Reply:
The nasty approach would be to leave most things in the example data package untyped, then rely on OUFR's build failing to catch mismatches...